### ISSUE TITLE
Hello! Can you publish the basic model of ViT-base? Or is it just a ViT-base distillation model?

### ISSUE THREAD

**Issue by DragonWang-cell** [OpenGVLab/VideoMAEv2#10]

**Comment by congee524** [comment]
Hi, for the same size ViT-variant we don't want to offer too many options, which might confuse our users.

You could perhaps use [internvideo's model](https://github.com/OpenGVLab/InternVideo/blob/main/Pretrain/VideoMAE/README.md). This model was pre-trained for 800 epochs on the same UnlabeledHybrid dataset without dual masking and is close to the performance of our VideoMAE V2-B which was pre-trained for 1200 epochs with dual masking.
